python-package/lightgbm/engine.py

```diff
@@ -460,7 +460,7 @@ def cv(params, train_set, num_boost_round=100,
         If list of int, interpreted as indices.
         If list of str, interpreted as feature names (need to specify ``feature_name`` as well).
         If 'auto' and data is pandas DataFrame, pandas unordered categorical columns are used.
-        All values in categorical features should be less than int32 max value (2147483647).
+        All values in categorical features will be cast to int32 and thus should be less than int32 max value (2147483647).
```

Comment:
One more place in `train()`:
https://github.com/microsoft/LightGBM/blob/717631af1836be9b03444ece485f3a3b5dbae5da/python-package/lightgbm/engine.py#L108